.github/workflows/ci.yml

```
@@ -47,9 +50,15 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version:  ${{ matrix.node-version }}
-      - name: Setup Foreman Plugin
-        uses: kamils-iRonin/setup-foreman-plugin@master
-        with:
-          plugin-name: ${{ env.FOREMAN_PLUGIN_NAME }}
-      - name: Run plugin tests
-        run: bundle exec rake test:${FOREMAN_PLUGIN_NAME}
+      - name: Setup Plugin
+        run: |
+          echo "gem 'foreman_git_templates', path: './foreman_git_templates'" > bundler.d/foreman_git_templates.local.rb
+          gem install bundler
+          bundle config set without journald development console mysql2 sqlite
+          bundle install --jobs=3 --retry=3
+          bundle exec rake db:create
+          bundle exec rake db:migrate
+          npm install
+          bundle exec rake webpack:compile
+      - name: Run Tests
```

```suggestion
      - name: Run plugin tests
```

Can we also add `bundle exec rake test TEST="test/unit/foreman/access_permissions_test.rb"` and `bundle exec rake "plugin:assets:precompile[${FOREMAN_PLUGIN_NAME}]" RAILS_ENV=production`